src/components/loader-star/loader-star.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Meta, StoryObj } from "@storybook/react";
+
+import LoaderStar from "./loader-star.component";
+
+const meta: Meta<typeof LoaderStar> = {
+  title: "Loader Star",
+  component: LoaderStar,
+  argTypes: {},
+  parameters: { chromatic: { disableSnapshot: true } },
```

Comment:
Messaged about this privately. Due to the animations involved in this, I decided against allowing the snapshots because I was concerned Chromatic might flag false positives if the snapshot is delayed by a millisecond.